pkg/secrethub/client.go

```diff
@@ -290,3 +291,34 @@ func (c *Client) userAgent() string {
 
 	return userAgent
 }
+
+// Resolve fetches the value of a secret, when the `ref` parameter has the
+// format `secrethub://<path>`. Otherwise it returns `ref` unchanged, as an array of bytes.
+func (c *Client) Resolve(ref string) ([]byte, error) {
+	bits := strings.Split(ref, "://")
```

Comment:
And also, what happens when you input SECRETHUB://path/to/secret. Is it and should it be case (in)sensitive?

Reply:
I made it case insensitive for now.